app/src/main/java/com/example/fundatecheroes/home/view/SplashActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.fundatecheroes.home.view
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+import com.example.fundatecheroes.R
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+        getSupportActionBar()?.hide()
+        val handle = Handler()
+        handle.postDelayed({ fazerLogin() }, 3000)
```

Comment:
Para remover o deprecated, nós podemos fazer dessa forma
```suggestion
                Handler(Looper.getMainLooper()).postDelayed({ chamarLoginActivity() }, 3000)
```